src/lib.rs

```diff
@@ -2,13 +2,16 @@ use hex::{FromHex, FromHexError};
 use plonky2::field::goldilocks_field::GoldilocksField;
 use plonky2::field::types::{Field, PrimeField64};
 use plonky2::hash::poseidon::PoseidonHash;
-use plonky2::plonk::config::Hasher;
+use plonky2::plonk::config::{Hasher, PoseidonGoldilocksConfig};
 use std::fmt;
 
 pub mod backend;
 pub mod frontend;
+pub mod merkletree;
 
 pub type F = GoldilocksField;
+pub type C = PoseidonGoldilocksConfig;
+pub const D: usize = 2;
```

Comment:
Can you write a doc line about what this value is?
For `C` and `D`, maybe use a more descriptive name?

Reply:
Also perhaps move this `D` and `C` to the merkletree file?  Do you think these type and constant need to be accessed generally?

Reply:
Cool, will add a comment to bring context; regarding moving them to the merkletree file, I think that not worth, since at least once we start adding circuits we will need them too.